superset-frontend/src/dashboard/stylesheets/dnd.less

```diff
@@ -123,7 +129,8 @@
   }
 
   & > .empty-droptarget:first-child {
-    height: 24px;
+    height: 48px;
```

Comment:
any reason why not to make this the entire size of the dashboard when it's empty?

Reply:
This is really good idea. But unfortunately it might not be that easy, because we would have to detect if first row is placed and dynamically change height of the `droptarget` element to limit back the area above first row.